### PR TITLE
[CORL-625] Create link details in the moderation card details tabs

### PR DESCRIPTION
**Pull request by nick-funk**
Adds copy-able link buttons to comment cards in the moderation queues in the details tabs.

## How to test

- Head to moderation queues
- Find a comment in the queues
- Click details on the comment card
- You should see two buttons to copy links
- Try them out, they link to single comment views for things

### CODE REVIEW

File: src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}) => {
+  return (
+    <>
+      <div className={styles.label}>
+        <Localized id="moderate-linkDetails-label">
+          Copy link to this comment
+        </Localized>
+      </div>
+      <div className={styles.buttonContainer}>
+        <CopyToClipboard text={`${comment.story.url}?commentID=${comment.id}`}>
```
Comment (wyattjoh) [inline src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx]: Should use `getURLWithCommentID` instead.

File: src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            <Icon size="md">link</Icon>
+            <Localized id="moderate-in-stream-link-copy">
+              <span>In Stream</span>
+            </Localized>
+          </Button>
+        </CopyToClipboard>
+      </div>
+      <div className={styles.buttonContainer}>
+        <CopyToClipboard
+          text={`${settings.organization.url}/admin/moderate/comment/${comment.id}`}
```
Comment (wyattjoh) [inline src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx]:
```suggestion
          text={`${getLocationOrigin()}/admin/moderate/comment/${comment.id}`}
```

**Review by wyattjoh — CHANGES_REQUESTED** [review]

File: src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx (hunk 1 above)
Comment (nick-funk) [reply src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx]: Makes sense!

File: src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx (hunk 2 above)
Comment (nick-funk) [reply src/core/client/admin/components/ModerateCard/LinkDetailsContainer.tsx]: Cool

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**